chef/cookbooks/neutron/templates/default/dnsmasq-neutron.conf.erb

```diff
@@ -0,0 +1,3 @@
+<% if @gre -%>
+dhcp-option-force=26,1458
+<% end -%>
```

Comment:
Maybe add a comment so that when gre is not used, this file is not totally empty and confusing?